File: applicatie/src/main/java/com/vdzon/administratie/afschrift/regelParsers/DefaultRegelParser.java
package com.vdzon.administratie.afschrift.regelParsers;

public class DefaultRegelParser implements RegelParser {

    @Override
    public boolean match(String omschrijving) {
        return false;
    }


    @Override
    public String extractNaam(String omschrijving) {
        return "Onbekend";
    }

    @Override
    public String extractOmschrijving(String omschrijving) {
        return "Onbekend";
    }
}

File: bankimport/impl/src/main/java/com/vdzon/administratie/abnamrobankimport/regelParsers/DefaultRegelParser.kt
package com.vdzon.administratie.abnamrobankimport.regelParsers

class DefaultRegelParser : RegelParser {
    override fun match(omschrijving: String): Boolean = false
    override fun extractNaam(omschrijving: String): String = "Onbekend"
    override fun extractOmschrijving(omschrijving: String) = "Onbekend"
}
